test/Microsoft.Health.Fhir.Tests.Integration/Features/Operations/Export/CreateExportRequestHandlerTests.cs

```diff
@@ -154,6 +155,22 @@ public async Task GivenDifferentDestination_WhenCreatingAnExportJob_ThenNewJobSh
             Assert.NotEqual(response.JobId, newResponse.JobId);
         }
 
+        [Fact]
+        public async Task GivenDestinationTypeOrDestinationConnectionSettings_WhenCreatingAnExportJob_ThenItShouldBeRemovedFromRequestUri()
+        {
+            const string baseUrl = "http://localhost/$export?_count=100";
+
+            var requestUri = new Uri($"{baseUrl}&_destinationType=type&_destinationConnectionSettings=settings");
```

Comment:
nit: might be nice to have one more param after the _destinationConnectionSettings to make sure that is still part of the url after the removal.